package.json

```diff
@@ -21,12 +21,9 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "@fal-works/esbuild-plugin-global-externals": "^2.1.2",
     "@types/node": "^18.11.19",
-    "@types/react": "^18.0.27",
```

Comment:
`@types/react` is required for linter and intellisense

Reply:
It's weird that the template plugin doesn't include that, so I thought it was useless, also because I didn't get any errors in vscode.